students/WooseokJ/Fixed/MailP4testWJFixed.py

```diff
@@ -0,0 +1,30 @@
+# import from the MailP4WJ file to test
+from MailP4WJFixed import add_donor
+from MailP4WJFixed import create_a_report_test
+from MailP4WJFixed import send_letters_to_all_donors_test
+
+donor_db = {"William Gates, III": [653784.49, 2],
+                "Mark Zuckerberg": [16396.10, 3],
+                "Jeff Bezos": [877.33, 1],
+                "Paul Allen": [708.42, 3],
+            }
+
+def test_send_a_thankyou1():
+    '''send a thankyou note to donor'''
+    expected = ('Paul Allen', [808.42, 4])
+    assert add_donor('Paul Allen', 100, donor_db) == expected
+
+
+def test_create_a_report():
+    '''test the text created by create_a_report'''
+    expected = "Mark Zuckerberg           $     16396.10             3  $     5465.37"
+    assert create_a_report_test('Mark Zuckerberg', [16396.10, 3]) == expected

```

Comment:
creating a separate function that doesn't run the code being run won't detect changes with anything you do in your `create_report` function